packages/cli/lib/sandbox-create.js

```diff
@@ -0,0 +1,337 @@
+const Spinnies = require('spinnies');
+const {
+  sandboxNamePrompt,
+  sandboxTypePrompt,
+} = require('./prompts/sandboxesPrompt');
+const {
+  sandboxTypeMap,
+  getSandboxLimit,
+  getHasSandboxesByType,
+  saveSandboxToConfig,
+  sandboxApiTypeMap,
+  getSandboxTypeAsString,
+  getAccountName,
+  STANDARD_SANDBOX,
+  DEVELOPER_SANDBOX,
+} = require('./sandboxes');
+const { i18n } = require('@hubspot/cli-lib/lib/lang');
+const { logger } = require('@hubspot/cli-lib/logger');
+const {
+  debugErrorAndContext,
+  logErrorInstance,
+} = require('@hubspot/cli-lib/errorHandlers/standardErrors');
+const { trackCommandUsage } = require('./usageTracking');
+const {
+  isMissingScopeError,
+  isSpecifiedError,
+} = require('@hubspot/cli-lib/errorHandlers/apiErrors');
+const { getHubSpotWebsiteOrigin } = require('@hubspot/cli-lib/lib/urls');
+const {
+  getEnv,
+  getAccountConfig,
+  getConfig,
+  getAccountId,
+} = require('@hubspot/cli-lib');
+const { createSandbox } = require('@hubspot/cli-lib/sandboxes');
+const { promptUser } = require('./prompts/promptUtils');
+const { syncSandbox } = require('./sandbox-sync');
+const {
+  setAsDefaultAccountPrompt,
+} = require('./prompts/setAsDefaultAccountPrompt');
+const { updateDefaultAccount } = require('@hubspot/cli-lib/lib/config');
+const { getValidEnv } = require('@hubspot/cli-lib/lib/environment');
+
+const i18nKey = 'cli.commands.sandbox.subcommands.create';
```

Comment:
Another small organization nit, but can this lang key be updated to point to the dedicated `cli.lib` section of the lang file?

Reply:
Good call! Will get these updated now